### PR TITLE
Update Russian translation to 2021-05-01

### PR DESCRIPTION
**Pull request by TimWCA**
Update Russian translations of WCA Regulations and Guidelines to ver. 2021-05-01.

### CODE REVIEW

File: russian/wca-guidelines.md [hunk 1]
```diff
@@ -131,6 +119,9 @@
 - 9f5+) [УТОЧНЕНИЕ] За попытку ставится результат DNS, если участник имел право сделать эту попытку, но не начал её (см. [Положение A3b2](regulations:regulation:A3b2)). Если участник не заслужил попытку (например, не уложился в лимит раунда с катоффом), то за эту попытку ему не ставится никакого результата.
 - 9p2+) [УТОЧНЕНИЕ] Здесь под результатом имеется в виду единичная попытка или среднее время, которое определяет место участника в соответсвующем раунде.
 - 9q+) [РЕКОМЕНДАЦИЯ] Рекомендуется, чтобы в дисциплинах и раундах участвовало не меньше 2 участников.
+- 9t+) [ОБЪЯСНЕНИЕ] Из-за [9t1](regulations:regulain:9t1) и [9t2](regulations:regulain:9t2) раунды могут выглядеть как несоответствующие [9p1](regulations:regulation:9p1), [9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) или [9m3](regulations:regulation:9m3). Однако внесение изменений для соответствия этим правилам затронет невиновных участников, поэтому результаты должны остаться прежними.
```
Comment (hanwu85) [inline russian/wca-guidelines.md:122]:
```suggestion
- 9t+) [ОБЪЯСНЕНИЕ] Из-за [9t1](regulations:regulation:9t1) и [9t2](regulations:regulation:9t2) раунды могут выглядеть как несоответствующие [9p1](regulations:regulation:9p1), [9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) или [9m3](regulations:regulation:9m3). Однако внесение изменений для соответствия этим правилам затронет невиновных участников, поэтому результаты должны остаться прежними.
```

File: russian/wca-guidelines.md [hunk 2]
```diff
@@ -131,6 +119,9 @@
...
+- 9t++) [ДОПОЛНЕНИЕ] Если изменения результатов происходят в течении соревнований, это правило также должно быть применено. В этом случае [9t1](regulations:regulain:9t1), [9t2](regulations:regulain:9t2) и [9t3](regulations:regulain:9t3) также должны быть соблюдены на усмотрение Делегата WCA.
```
Comment (hanwu85) [inline russian/wca-guidelines.md:123]: typo from the original version.
```suggestion
- 9t++) [ДОПОЛНЕНИЕ] Если изменения результатов происходят в течении соревнований, это правило также должно быть применено. В этом случае [9t1](regulations:regulation:9t1), [9t2](regulations:regulation:9t2) и [9t3](regulations:regulation:9t3) также должны быть соблюдены на усмотрение Делегата WCA.
```

Comment (TimWCA) [reply russian/wca-guidelines.md]: Thank you :)

File: russian/wca-guidelines.md (hunk 1 above)
Comment (TimWCA) [reply russian/wca-guidelines.md]: :)

File: russian/wca-guidelines.md [hunk 3]
```diff
@@ -131,6 +119,9 @@
 - 9f5+) [УТОЧНЕНИЕ] За попытку ставится результат DNS, если участник имел право сделать эту попытку, но не начал её (см. [Положение A3b2](regulations:regulation:A3b2)). Если участник не заслужил попытку (например, не уложился в лимит раунда с катоффом), то за эту попытку ему не ставится никакого результата.
 - 9p2+) [УТОЧНЕНИЕ] Здесь под результатом имеется в виду единичная попытка или среднее время, которое определяет место участника в соответсвующем раунде.
 - 9q+) [РЕКОМЕНДАЦИЯ] Рекомендуется, чтобы в дисциплинах и раундах участвовало не меньше 2 участников.
+- 9t+) [ОБЪЯСНЕНИЕ] Из-за [9t1](regulations:regulation:9t1) и [9t2](regulations:regulation:9t2) раунды могут выглядеть как несоответствующие [9p1](regulations:regulation:9p1), [9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) или [9m3](regulations:regulation:9m3). Однако внесение изменений для соответствия этим правилам затронет невиновных участников, поэтому результаты должны остаться прежними.
```
Comment (AlexKatyukov) [inline russian/wca-guidelines.md:122]: Hi! This is a review that Timofei asked me to do :) It's in Russian (that's logic)

Мне не очень нравится слово «невиновных». Оно отражает смысл, но какое-то оно странное в данном контексте.

File: russian/wca-guidelines.md [hunk 4]
```diff
@@ -131,6 +119,9 @@
...
+- 9t++) [ДОПОЛНЕНИЕ] Если изменения результатов происходят в течении соревнований, это правило также должно быть применено. В этом случае [9t1](regulations:regulation:9t1), [9t2](regulations:regulation:9t2) и [9t3](regulations:regulation:9t3) также должны быть соблюдены на усмотрение Делегата WCA.
```
Comment (AlexKatyukov) [inline russian/wca-guidelines.md:123]: В течени**е**

File: russian/wca-regulations.md [hunk 5]
```diff
@@ -495,14 +504,15 @@
     - E3c) Наклейки (свои).
     - E3d) Часы или секундомер (свои) для слежения за оставшимся временем, если Делегат WCA разрешил их использование.
     - E3e) Другие неэлектронные средства, которые не дадут несправедливого преимущества участнику по усмотрению Делегата WCA (см. [Положение 2i1](regulations:regulation:2i1)).
+    - E3f) Инструменты для удаление пометок (принесённый с собой), такие как корректирующая жидкость, корректирующая лента или ластики.
```
Comment (AlexKatyukov) [inline russian/wca-regulations.md:507]: Удалени**я**

Comment (AlexKatyukov) [reply russian/wca-regulations.md:507]: И, наверное, принесённы**е**

**Review by AlexKatyukov — CHANGES_REQUESTED** [review]
Some not critical comments to Timofei

**Comment by Ivan-Brigidano** [comment]
We are still having some software issues with Travis :/   The issues are not related to the PR but to Travis itself. The WST and the WRC are working to solve them ASAP.
Other than this, please tell us when the PR is ready so that we can merge it.

**Comment by TimWCA** [comment]
I think, it is ready to merge :)

**Comment by TimWCA** [comment]
> We are still having some software issues with Travis :/ The issues are not related to the PR but to Travis itself. The WST and the WRC are working to solve them ASAP.
> Other than this, please tell us when the PR is ready so that we can merge it.

**Comment by TimWCA** [comment]
I'm sorry, I accidentally closed this PR :) I reopened it.

**Review by Ivan-Brigidano — APPROVED** [review]

**Pull request merged**